.github/workflows/interop.yml

```diff
@@ -46,3 +50,7 @@ jobs:
         with:
           test-filter: nim-libp2p-head
           extra-versions: ${{ github.workspace }}/tests/hole-punching-interop/version.json
+          s3-cache-bucket: ${{ vars.S3_LIBP2P_BUILD_CACHE_BUCKET_NAME }}
+          s3-access-key-id: ${{ vars.S3_LIBP2P_BUILD_CACHE_AWS_ACCESS_KEY_ID }}
+          s3-secret-access-key: ${{ secrets.S3_LIBP2P_BUILD_CACHE_AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ vars.S3_LIBP2P_BUILD_CACHE_AWS_REGION }}
```

Comment:
would it be possible to avoid the duplication?

Reply:
Oh, in both calls, you mean. Not that I know.

Reply:
something like this might work

```
name: Interoperability Tests

on:
  pull_request:
  merge_group:
  push:
    branches:
      - master
  workflow_dispatch:

concurrency:
  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
  cancel-in-progress: true

jobs:
  run-interop-tests:
    runs-on: ubuntu-22.04
    strategy:
      matrix:
        test-type: 
          - transport-interop
          - hole-punching-interop

    name: Run ${{ matrix.test-type }} Tests
    
    steps:
      - name: Free Disk Space (Ubuntu)
        uses: jlumbroso/free-disk-space@v1.3.1
        with:
          tool-cache: true

      - name: Checkout Code
        uses: actions/checkout@v4

      - name: Setup Docker Buildx
        uses: docker/setup-buildx-action@v3

      - name: Build Docker Image
        run: |
          docker buildx build --load -t nim-libp2p-head -f tests/${{ matrix.test-type }}/Dockerfile .

      - name: Run Tests
        uses: libp2p/test-plans/.github/actions/run-${{ matrix.test-type == 'transport-interop' && 'transport-interop-test' || 'interop-hole-punch-test' }}@master
        with:
          test-filter: nim-libp2p-head
          extra-versions: ${{ github.workspace }}/tests/${{ matrix.test-type }}/version.json
          s3-cache-bucket: ${{ vars.S3_LIBP2P_BUILD_CACHE_BUCKET_NAME }}
          s3-access-key-id: ${{ vars.S3_LIBP2P_BUILD_CACHE_AWS_ACCESS_KEY_ID }}
          s3-secret-access-key: ${{ secrets.S3_LIBP2P_BUILD_CACHE_AWS_SECRET_ACCESS_KEY }}
          aws-region: ${{ vars.S3_LIBP2P_BUILD_CACHE_AWS_REGION }}
```

Reply:
I personally feel it's clearer the other way, so for me the tradeoff is not worth. 
If you still want to go with it please confirm and I'll amek the change.

Reply:
I kinda agree with @AlejandroCabeza here. Sometimes, trying to remove duplicates at all cost is detrimental to understanding the code.

Reply:
I'll wait for an agreement on this one.

Reply:
To be honest, it's not a very strong feeling on my part, we can change if Diego is adamant about it.